### PR TITLE
feat: document form HTML component

### PR DESCRIPTION
**Pull request by Skaiir**
## Description

We are adding a new component to forms which can render HTML with very little restriction, provided it's not interactive (so no javascript)

## When should this change go live?

- [x] This change is not yet live and should not be merged until 8.5.alpha1.
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/forms/form-element-library/forms-element-library-html.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+description: A form element to display HTML content.
+---
+
+A flexible display component designed to quickly render HTML content for the user.
+
+![Form HTML Symbol](/img/form-icons/form-html.svg)
+
+## Configurable properties
+
+- **Content**: This property accepts HTML content. It can be defined as an [expression](../../feel/language-guide/feel-expressions-introduction.md), using [templating syntax](../configuration/forms-config-templating-syntax.md) or as plaintext HTML. Javascript and `<style>` tags are purposefully sanitized out for the sake of security, inline styles are however perfectly valid.
```
Comment (christian-konrad) [inline docs/components/modeler/forms/form-element-library/forms-element-library-html.md:13]:
```suggestion
- **Content**: This property accepts HTML content. Define it using [templating syntax](../configuration/forms-config-templating-syntax.md) or as plaintext HTML. Javascript and `<style>` tags are purposefully sanitized out for the sake of security. Use inline styles instead, and templating to show dynamic content based on form context variables.
```

File: docs/components/modeler/forms/form-element-library/forms-element-library-html.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- **Hide if**: [Expression](../../feel/language-guide/feel-expressions-introduction.md) to conditionally hide the HTML content.
+- **Columns**: Space the field will use inside its row. The **Auto** means it will automatically adjust to available space in the row. Read more about the underlying grid layout in the [Carbon Grid documentation](https://carbondesignsystem.com/guidelines/2x-grid/overview).
+
+## HTML configurations
+
+These examples demonstrate how to use HTML to display content. JavaScript and `<style>` tags are entirely filtered out of the rendered output, which happens after any templating or expression evaluation. This allows this component to be very flexible in terms of display use cases without exposing any security risks such as XSS. Styling in-line via style attributes is allowed.
```
Comment (christian-konrad) [inline docs/components/modeler/forms/form-element-library/forms-element-library-html.md:19]: I'd be careful here to not overpromise:

```suggestion
These examples demonstrate how to use HTML to display content. JavaScript and `<style>` tags are entirely filtered out of the rendered output, which happens after any templating or expression evaluation. This allows this component to be very flexible in terms of display use cases without exposing XSS security risks. Styling in-line via style attributes is allowed.
```

**Review by christian-konrad — DISMISSED** [review]
2 small changes requested

File: docs/components/modeler/forms/form-element-library/forms-element-library-html.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+<ul>
+  <li>First Item</li>
+  <li>Second Item</li>
+</ul>
+
+<img src="https://someurl.com/image.png" alt="Descriptive Image Text">
+
+```
+
+**Templating Notation**
```
Comment (christinaausley) [inline docs/components/modeler/forms/form-element-library/forms-element-library-html.md:52]: Nitpick, but we can make these sentence-case, so "Templating notation" for example.

File: docs/components/modeler/forms/form-element-library/forms-element-library-text.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
 id: forms-element-library-text
 title: Text view
-description: A form element to display static information.
+description: A form element to display simple markdown-powered text.
 ---
 
-A text component allowing to display static information to the user.
+A markdown powered text component allowing to display simple information to the user.
```
Comment (christinaausley) [inline docs/components/modeler/forms/form-element-library/forms-element-library-text.md:7]: Markdown, capitalized 👍  So, Markdown-powered.

**Review by christinaausley — COMMENTED** [review]
Really nice writing here, and only a few nitpicks 👍

File: docs/components/modeler/forms/form-element-library/forms-element-library-html.md (hunk 2 above)
Comment (christinaausley) [reply docs/components/modeler/forms/form-element-library/forms-element-library-html.md:19]: This is a great callout.

File: docs/components/modeler/forms/form-element-library/forms-element-library-text.md (hunk 4 above)
Comment (christian-konrad) [reply docs/components/modeler/forms/form-element-library/forms-element-library-text.md:7]:
```suggestion
A Markdown-powered text component allowing to display simple information to the user.
```

File: docs/components/modeler/forms/form-element-library/forms-element-library-html.md (hunk 2 above)
Comment (Skaiir) [reply docs/components/modeler/forms/form-element-library/forms-element-library-html.md]: The new version's text was changed. This should no longer be a problem.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**